src/main/client/src/components/app/table/react-table/ReactTableContainer.js

```diff
@@ -56,6 +58,9 @@ const mapDispatchToProps = dispatch => ({
     addFiltered: (filtered) => {
         dispatch(addFiltered(filtered));
     },
+    setActivePage: (page, layerId) => {
```

Comment:
Funktiota kutsutaan yhdellä parametrilla, joka on objekti, joten layerId on turha. Vaikuttaa myös seuraavaan kommenttiin. Vaihtoehtoisesti voisi pitää kahdella parametrilla kutsuttavana ja muuttaa muut paikat. Tämä olisi mielestäni sinällään loogisempi, mutta toimii näinkin.
```suggestion
    setActivePage: (page) => {
```